src/librustc_trans/type_.rs

```diff
@@ -268,6 +269,20 @@ impl Type {
         }
     }
 
+    pub fn fat_ptr_data_type(&self) -> Type {
+        unsafe {
+            Type::from_ref(llvm::LLVMStructGetTypeAtIndex(self.to_ref(),
+                                                          abi::FAT_PTR_ADDR as u32))
+        }
+    }
+
+    pub fn fat_ptr_extra_type(&self) -> Type {
```

Comment:
`unsized_info_ty` it seems. Anyways, didn't actually use this. Just added it for completeness sake but looks like it's also unneeded.